test/unit/api/python/test_symbol_manager.py

```diff
@@ -23,9 +23,14 @@ def tm():
 def solver(tm):
     return cvc5.Solver(tm)
 
+def constructor(tm, solver):
```

Comment:
Test function names must start with the prefix `test_` to be executed by pytest.
```suggestion
def test_constructor(tm, solver):
```